cmd/agent/gui/agent.go

```diff
@@ -149,7 +149,7 @@ func makeFlare(w http.ResponseWriter, r *http.Request, flare flare.Component) {
 		return
 	}
 
-	res, e := flare.Send(filePath, payload.CaseID, payload.Email)
+	res, e := flare.Send(filePath, payload.CaseID, payload.Email, "")
```

Comment:
Would not makes sense to use source to pass values like this "local-gui" or "local-cmd"?